MediaBrowser.Providers/Music/AudioMetadataService.cs

```diff
@@ -40,6 +41,45 @@ protected override void MergeData(MetadataResult<Audio> source, MetadataResult<A
             {
                 targetItem.Album = sourceItem.Album;
             }
+
+            if (replaceData || string.IsNullOrEmpty(targetItem.GetProviderId(MetadataProvider.MusicBrainzAlbumArtist)))
+            {
+                var targetAlbumArtistId = targetItem.GetProviderId(MetadataProvider.MusicBrainzAlbumArtist);
```

Comment:
```suggestion
```
repeat for all (split for readability)

Reply:
I don't really understand this suggestion?